test/Serilog.Sinks.MSSqlServer.Tests/Sinks/MSSqlServer/Output/AdditionalColumnDataGeneratorTests.cs

```diff
@@ -202,5 +202,28 @@ public void GetAdditionalColumnNameAndValueReturnsTruncatedForCharacterTypesWith
             Assert.Equal(columnName, result.Key);
             Assert.Equal("Additio...", result.Value);
         }
+
+        [Fact]
```

Comment:
A new test in `StringExtensionsTests` would be worthwhile.

Reply:
Ah, I missed that, I'll add one today after work.

Reply:
Alright I have added a new test in StringExtensionsTests now. @jonorossi